generic_cpe.xml.fdm_material

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-Generic CPE profile. Serves as an example file, data in this file is not correct.
+Generic CPE profile. The data in this file may not be correct for your specific machine.
```

Comment:
I like this change. Originally it was meant as an example file but this is not quite right any more. Thanks.